elasticdl/python/master/optimizer_wrapper.py

```diff
@@ -100,14 +100,16 @@ def _var_key(var):
 class OptimizerWrapper(object):
     """ ElasticDL optimizer wrapper.
 
-    If model does not use ElasticDL embedding layer, `OptimizerWrapper`
+    If model does not use ElasticDL embedding layer, OptimizerWrapper
```

Comment:
Why changing this line? You can keep them since you are referring a particular class

Reply:
Done.